payment-cards/declines.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: Card Declines
+permalink: /payment-cards/declines/
+---
+
+#Card Declines
```

Comment:
How do you feel about `#Header` vs. `# Header`? I personally prefer the latter but I don't have super strong feelings about it. Just good to be consistent.